tree/ntuple/v7/test/ntuple_schemaext.cxx

```diff
@@ -0,0 +1,243 @@
+#include "ntuple_test.hxx"
+
+TEST(RNTuple, SchemaExtensionSimple)
```

Comment:
The test (and file) might be misnamed.

Reply:
Thanks for noticing :+1: .  I have added a commit to https://github.com/root-project/root/pull/12676 that renames the test.

Reply:
Thanks.